File: tourneyAPI/Models/Entities/Characters/BanjoKazooie.cs
namespace Entities;

using Enums;


// Defines the competitive profile metadata for this playable character.
public class BanjoKazooie : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public BanjoKazooie()
    {
        Id = CharacterId.BanjoKazooie;
        characterName = CharacterName.BANJO_AND_KAZOOIE;
        archetype = Archetype.TURTLE;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.HEAVYWEIGHT;
        tierPlacement = TierPlacement.C;
    }

}

File: tourneyClient/src/models/entities/Characters/BanjoKazooie.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const BanjoKazooie: Character =
{
    id: CharacterId.BanjoKazooie,
    characterName: CharacterName.BANJO_AND_KAZOOIE,
    archetype: Archetype.TURTLE,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.HEAVYWEIGHT,
    tierPlacement: TierPlacement.C
}

export default BanjoKazooie; 